ntoskrnl/io/iomgr/deviface.c

```diff
@@ -451,12 +451,13 @@ IopOpenInterfaceKey(IN CONST GUID *InterfaceClassGuid,
  *--*/
 NTSTATUS
 NTAPI
-IoGetDeviceInterfaces(IN CONST GUID *InterfaceClassGuid,
+IopGetDeviceInterfacesWorker(IN CONST GUID* InterfaceClassGuid,
                       IN PDEVICE_OBJECT PhysicalDeviceObject OPTIONAL,
                       IN ULONG Flags,
-                      OUT PWSTR *SymbolicLinkList)
+                      IN BOOLEAN UseKernelPath,
+                      OUT PWSTR* SymbolicLinkList)
```

Comment:
```suggestion
    _In_ BOOLEAN UseKernelPath,
    _Outptr_result_nullonfailure_
    _At_(*SymbolicLinkList, _When_(return==0, __drv_allocatesMem(Mem)))
        PZZWSTR *SymbolicLinkList)
```

Reply:
OK I had to specify these suggestions by parts, since github PR interface is a bit retard when one tries to specify suggestions across deleted lines.